domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error/DecreasingStopTimeDistanceErrorNotice.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (c) 2020. MobilityData IO.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.notice.error;
+
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.NoticeExporter;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.base.ErrorNotice;
+
+import java.io.IOException;
+
+public class DecreasingStopTimeDistanceErrorNotice extends ErrorNotice {
+    public DecreasingStopTimeDistanceErrorNotice(final String tripId,
+                                                 final int stopSequence,
```

Comment:
I would rename these variables to define the order of sequence, like:
* `stopSequence`
* `shapeDistTraveled`
* `previousStopSequence`
* `previousShapeDistTraveled`

Per my comment below, without explicit order in the names it would be easy to accidentally swap the variables when calling the method, unless you looked at the implementation.